### DISCUSSION TITLE
无光之海船长无法创建

### DISCUSSION THREAD

**Discussion by lnxyz** [Gamuxorg/bbs#310] [游戏或网站运行反馈] [answered]
### 以下是推荐模板
游戏名称：无光之海

操作系统：ubuntu 19.10

bug描述：运行正常，但无法创建船长，没有船长船不能航行。

**Comment by shouhuanxiaoji** [comment]
已确认该问题，之前测试时以为是操作不对，原来是船长创建问题。

**Comment by lnxyz** [comment]
> 已确认该问题，之前测试时以为是操作不对，原来是船长创建问题。

有一个原因，是可能汉化造成的。
现在是船官里面没有人物，所以根本添加不了船员。船只就不能航行。

**Comment by shouhuanxiaoji** [comment]
明白了 我们会想办法修复一下

**Comment by shouhuanxiaoji** [comment, marked as answer]
修复了一下，可以正常创建船长并开船，但代价是菜单无法汉化，这会导致你说的问题。现在事件是汉化的，请给予我们更多建议，感谢
